pkg/auth/audit/handler.go

```diff
@@ -59,12 +59,15 @@ func (h handler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 
 		statusCode: http.StatusOK,
 	}
+	body, userName := copyReqBody(req)
 	h.next.ServeHTTP(wr, req)
 	wr.Apply()
 
 	respTimestamp := time.Now().Format(time.RFC3339)
 
 	log := newLog(user, req, wr, reqTimestamp, respTimestamp)
+	log.setUserName(userName)
+	log.setBody(body)
```

Comment:
Might be cleaner to just pass these as args instead of the separate mutater funcs